packaging/debian-control/libnetwork-prerm.sh

```diff
@@ -6,5 +6,6 @@ set -e
 /etc/init.d/libnetwork stop
 
 update-rc.d -f  libnetwork remove
-rm -rf /opt/pg/libnetwork/
+rm -rf /opt/pg/libnetwork/config.ini
+rm -rf /opt/pg/libnetwork/libnetwork
```

Comment:
How are we planning to handle upgrade? Have we tested it? Standard DEB upgrade works as follow:
- extract content of new package
- prerm of old package
- postinst of new package.
I am afraid with this we will end up cleaning config and binaries on each upgrade.